tsconfig.json

```diff
@@ -5,5 +5,9 @@
     { "path": "./packages/nbdime" },
     { "path": "./packages/webapp" },
     { "path": "./packages/labextension" }
-  ]
+  ],
+
+  "compilerOptions": {
+    "types": ["node"]
```

Comment:
Previously, "types" had been left undefined on purpose, because TS would then automatically resolve which types it needed. Is this no longer the case?

Reply:
`compilerOptions` will be removed.